components/Restaurant_card.tsx

```diff
@@ -0,0 +1,61 @@
+import Restaurant from "@/models/Restaurant";
+import Link from "next/link";
+
+interface ResCardProps {
+  restaurant: Restaurant;
+}
+
+export default function Restaurant_card({ restaurant }: ResCardProps) {
```

Comment:
Nit: Feel free to refactor to be `RestaurantCard` for convention sake!

Reply:
Fixed!